### PR TITLE
fix openshift/origin/issues/15410.  Add oc into origin-ansible image

### PR DESCRIPTION
**Pull request by jcantrill**
This PR add oc back to the image for 'oc cluster up' to fix openshift/origin/issues/15410

### CODE REVIEW

**Comment by jcantrill** [comment]
aos-ci-test

File: images/installer/Dockerfile [hunk 1]
```diff
@@ -4,8 +4,10 @@ MAINTAINER OpenShift Team <dev@lists.openshift.redhat.com>
 
 USER root
 
+ADD images/installer/origin.repo /etc/yum.repos.d/
```
Comment (sosiouxme) [inline images/installer/Dockerfile]: @jcantrill Actually could you just add the repo definition under images/installer/root/etc/yum.repos.d and move this section to the top:

```
# Add image scripts and files for running as a system container
COPY images/installer/root /
```

Comment (sosiouxme) [inline images/installer/Dockerfile]: no wait, scratch that. don't want it in OCP.

**Comment by sosiouxme** [comment]
I would prefer to merge https://github.com/openshift/openshift-ansible/pull/5003 instead. I feel better about providing a smaller, signed, released client RPM.

**Pull request closed without merging**